test/v1/driver.test.js

```diff
@@ -64,7 +64,7 @@ describe('driver', () => {
 
   it('should handle connection errors', done => {
     // Given
-    driver = neo4j.driver("bolt://localhoste", sharedNeo4j.authToken);
+    driver = neo4j.driver("bolt://localhost", sharedNeo4j.authToken);
```

Comment:
@jsoref this spelling error is intentional. Test verifies connection errors. Build is red because this test fails.

Reply:
Would "local-host" suffice?

Reply:
I think yes